pkg/internal/cnsoperator/cnsfilevolumeclient/cnsfilevolumeclient.go

```diff
@@ -67,7 +66,7 @@ func InitFileVolumeClients(ctx context.Context) (*FileVolumeClients, error) {
 	}
 	store := informer.Informer().GetStore()
 	stopCh := make(chan struct{})
-	informer.Informer().Run(stopCh)
+	go informer.Informer().Run(stopCh)
 	if !cache.WaitForCacheSync(stopCh, informer.Informer().HasSynced) {
```

Comment:
Do we need to  sync the cache before invoking Run? Is there a possibility that we do not have something in the cache yet when AddClientVMToIPList is invoked?

Reply:
No i dont think. The caller will wait for `InitFileVolumeClients` to return before having the opportunity to call `AddClientVMToIPList` or `RemoveClientVMFromIPList`.